src/seedu/addressbook/Main.java

```diff
@@ -4,16 +4,21 @@
 import java.util.List;
 import java.util.Optional;
 
+
+import seedu.addressbook.commands.AddCommand;
 import seedu.addressbook.commands.Command;
 import seedu.addressbook.commands.CommandResult;
 import seedu.addressbook.commands.ExitCommand;
+
+import seedu.addressbook.commands.*;
```

Comment:
This is a coding style violation, you need to specify the exact class/interface imported.